logger/logger.go

```diff
@@ -0,0 +1,42 @@
+// Based on https://betterstack.com/community/guides/logging/zerolog/
+
+package logger
+
+import (
+	"io"
+	"os"
+	"runtime/debug"
+	"strconv"
+	"sync"
+
+	"github.com/rs/zerolog"
+)
+
+var once sync.Once
+var log zerolog.Logger
+
+func Get() zerolog.Logger {
+	once.Do(func() {
+		var logLevel int8
+		parsed, err := strconv.ParseInt(os.Getenv("LOG_LEVEL"), 10, 8)
+		if err != nil {
+			logLevel = int8(zerolog.InfoLevel) // Default to INFO
+		} else {
+			logLevel = int8(parsed)
+		}
+
+		var output io.Writer = zerolog.ConsoleWriter{
+			Out: os.Stdout,
+		}
+
+		buildInfo, _ := debug.ReadBuildInfo()
+		log = zerolog.New(output).
+			Level(zerolog.Level(logLevel)).
+			With().
+			Timestamp().
```

Comment:
[q] can we use a human readable timestamp here? 
Based on the docs it seems like the default format is unix epoch

Reply:
Hmmm interesting. I'm actually seeing just a time in my console logs. Though I suppose the current format isn't terribly useful. For example:

> 2:32PM WRN No Slack token found. Skipping communication. go_version=go1.19.7

It would probably be better to have something like

> 2023-03-27 10:04:00-07

Let me see what I can do about that...